css-fonts/css-fonts-quiz-notes.md

```diff
@@ -5,9 +5,9 @@
 After completing this exercise, you should be able to discuss or answer the following questions:
 
 - Name two types of units that can be used to adjust `font-size` in CSS.
-
+Font family and font size.
```

Comment:
It's asking what type of units you can use to adjust the actual `font-size` of an element. So something like `px`.